cmd/oci-image-tool/validate.go

```diff
@@ -121,6 +121,12 @@ func validatePath(name string) error {
 		return schema.ValidatorMediaTypeImageIndex.Validate(f)
 	case image.TypeConfig:
 		return schema.ValidatorMediaTypeImageConfig.Validate(f)
+	case "JSON":
```

Comment:
I don't think we introduce a new type `JSON` is a good idea here, `JSON` is not any type in image spec. I think it's better to keep the old way.

Reply:
I also think the previous method is good, and I have not thought of other methods at present, but the suggestion of @stevvooe  is to replace the previous method.